src/test/resources/solr/dssolr/conf/schema.xml

```diff
@@ -403,6 +404,44 @@ In solrconfig.xml the classic schema is defined with this:  <schemaFactory class
     <?example     Children?>
   </field>
 
+  <!-- Fields that only apply to tv metadata -->
+
+  <field name="original_title" type="string" indexed="true" stored="true">
```

Comment:
`string` is indexed and stored by default. This (and multiple similar fields) could just be `<field name="original_title" type="string">`.

...And we really need to talk about our schema in general, but that is a matter for another session.